frontend/javascripts/messages.tsx

```diff
@@ -254,6 +254,10 @@ instead. Only enable this option if you understand its effect. All layers will n
     'All trees are currently hidden. You can disable this by toggling the "Skeleton" layer in the layer settings in the left sidebar.',
   "tracing.invalid_json_url_hash":
     "Cannot parse JSON URL hash. More information was printed to the browser's console.",
+  "tracing.pin_mapping_info":
+    "The active volume annotation layer has an active mapping. Once you start mutating the layer, the mapping will be permanently enabled. This can only be undone by restoring an older version of this annotation. Are you sure you want to continue?",
```

Comment:
```suggestion
    "The active volume annotation layer has an active mapping. By mutating the layer, the mapping will be permanently pinned and can no longer be changed or disabled. This can only be undone by restoring an older version of this annotation. Are you sure you want to continue?",
```

Maybe it's easier to understand like this?

Reply:
I was also thinking about that which is why I added the "and can no longer be changed or disabled" explaining bit. To me "permanently enabled" sounds somewhat ambiguous and doesn't make clear that the mapping can no longer be changed.
So there's only one possible solution, I asked our future AI overlords which came up with these:

```
"By mutating the layer, the mapping will be permanently fixed in place (and can no longer be changed or disabled)."
"By mutating the layer, the mapping will be permanently locked (and can no longer be changed or disabled)."

These alternatives may help ensure that the meaning is clear to a broader audience.
```

which I both like. @philippotto What do you think?

Reply:
I think »locked« is good. We could even consider using that word in code as well instead of pinned? Then we don’t have this disparity between the UI and the code. That would be an undertaking, though 😅

Reply:
I'm also in favor of locked :+1: 

> We could even consider using that word in code as well instead of pinned? Then we don’t have this disparity between the UI and the code. That would be an undertaking, though 😅

I think, in the front-end it would be a simple search&replace. So, it's your call if it's worth the effort in the back-end :)

Reply:
Yeah, it’s not that much in the backend either. If we want to do it, we should do it in this PR, though, because changing the json update actions in the fossildb later would not be fun.

Reply:
Ok done, I hope I renamed every occurrence of "pinned" / "pin".